starfish/pipeline/filter/white_tophat.py

```diff
@@ -0,0 +1,34 @@
+from ._base import FilterAlgorithmBase
+
+
+class WhiteTophatFilter(FilterAlgorithmBase):
```

Comment:
I think having Filter in the name is extraneous. WhiteTophatFilter -> WhiteTophat? Same goes for Gaussian filter(s)